src/partials/head-meta-opengraph.hbs

```diff
@@ -1,6 +1,12 @@
-<meta property="og:locale" content="en_us" />
-<meta property="og:type" content="article" />
-<meta property="og:title" content="{{page.title}}" />
-<meta property="og:description" content="{{page.description}}" />
-<meta property="og:url" content="{{page.canonicalUrl}}" />
-<meta property="og:site_name" content="{{#if site.title}}{{site.title}}{{/if}}" />
+    <meta property="og:locale" content="en_us" />
```

Comment:
Indentation looks wrong

Reply:
yes, this may look strange, but see if we do not do this:
```
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
...
<meta property="og:image" content="https://doc.owncloud.com/opengraph-default.png" />
<meta property="og:image:alt" content="The Branded Clients documentation" />
```
I did for better readability of the content!